### DISCUSSION TITLE
errro 500

### DISCUSSION THREAD

**Discussion by moiiom59** [dbarzin/mercator#565] [General]
Je me retrouve avec des erreur 500 sur  Outils>Exploration.
Voici la copie ecran et les logs
![Capture d'écran 2023-12-01 162300](https://github.com/dbarzin/mercator/assets/43005914/9c5c8a41-4d96-4213-ab52-e19400bf5b92)
[laravel.log](https://github.com/dbarzin/mercator/files/13543237/laravel.log)
merci à vous

**Comment by moiiom59** [comment]
Sur les rapports 
![Capture d'écran 2023-12-04 084424](https://github.com/dbarzin/mercator/assets/43005914/24bb8376-d73e-4f58-ae56-d7c2e995c82f)
![Analyse des besoins de sécurité_Capture d'écran 2023-12-04 084240](https://github.com/dbarzin/mercator/assets/43005914/9cb752d5-8ef7-4879-b5e8-a7b81e1414cb)
![Application par groupe applicatif_Capture d'écran 2023-12-04 084145](https://github.com/dbarzin/mercator/assets/43005914/1b4d8f0a-8e17-4d56-ae56-23d960640467)

**Comment by moiiom59** [comment]
J'ai bien fait la mise a jour et rebooter la machine mais le problème persiste
Le fichier RepoprtControler.php a bien été mise à jour !
![Git_Capture d'écran 2023-12-04 095949](https://github.com/dbarzin/mercator/assets/43005914/5b598c90-9cca-49da-b042-9978fd5b4cd5)

**Comment by dbarzin** [comment]
Pouvez-vous envoyer le log de l'erreur ?

**Comment by moiiom59** [comment]
![err_Capture d'écran 2023-12-04 101549](https://github.com/dbarzin/mercator/assets/43005914/081a1985-c871-4bb3-a19b-bf9e71fd4d27)

**Comment by moiiom59** [comment]
[laravel.log](https://github.com/dbarzin/mercator/files/13544432/laravel.log)
Le dernier log

**Comment by moiiom59** [comment]
Vous avez bien accès au dernier logs sur le forum ?

Bien cordialement,

**Comment by dbarzin** [comment]
OK, corrigé ici : https://github.com/dbarzin/mercator/pull/567
